protocol-units/execution/maptos/framework/releases/README.md

```diff
@@ -13,4 +13,37 @@ We use this directory to track framework releases and migrations. The path of mi
     4. Implement the new release in the new directory.
     5. Write a migration test for it against the named release which is now in the [`parent`](./parent/) directory.
     6. Update the [`latest`](./latest/) directory to re-export the new release.
-- The [`head`](./head/) crate is a special case that is used to track the release that is on the current branch head. It can be used for intermediately testing changes that are not yet ready to be published.
\ No newline at end of file
+- The [`head`](./head/) crate is a special case that is used to track the release that is on the current branch head. It can be used for intermediately testing changes that are not yet ready to be published.
+
+# Feature migration
+
+The feature comparission between MVT Network and Aptos Network gives these features that are different.
+
+| Title                                      | MVT     | APTOS   | MIGRATED Network |
+|--------------------------------------------|---------|---------|------------------|
+| APTOS_STD_CHAIN_ID_NATIVES                 | ENABLE  | DISABLE |                  |
+| PERIODICAL_REWARD_RATE_DECREASE            | DISABLE | ENABLE  |                  |
+| PARTIAL_GOVERNANCE_VOTING                  | DISABLE | ENABLE  |                  |
+| DELEGATION_POOL_PARTIAL_GOVERNANCE_VOTING  | DISABLE | ENABLE  |                  |
+| VM_BINARY_FORMAT_V7                        | DISABLE | ENABLE  | DISABLE          |
+| KEYLESS_BUT_ZKLESS_ACCOUNTS                | ENABLE  | DISABLE |                  |
```

Comment:
this should be disabled `KEYLESS_BUT_ZKLESS_ACCOUNTS`

Reply:
Updated

Reply:
Done